assets/diagrams/promote.mmd

```diff
@@ -0,0 +1,13 @@
+graph LR
+A(npm dist-tag add)
+B(fa:fa-server warehouse.ai)
+C{fa:fa-database No-SQL Database}
+D{fa:fa-database Artifactory}
```

Comment:
`Artifactory` should probably be something like `NPM-like registry`

Reply:
good call